CHANGELOG.md

```diff
@@ -6,81 +6,83 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 ### Added
- - Empty implementation of the cron action
- - Shopgate config variables to DI to support app:config:dump call
+- Empty implementation of the cron action
+- Shopgate config variables to DI to support app:config:dump call
+- App-only cart rules
```

Comment:
add the pointer that it is for CE only OR that it does not work with CustomerSegment enabled.